frontend/projects/upgrade/src/app/shared/services/common-dialog.service.ts

```diff
@@ -156,7 +159,7 @@ export class DialogService {
       data: commonModalConfig,
       width: '656px',
       height: 'auto',
-      autoFocus: 'input',
+      autoFocus: false,
```

Comment:
what element does it focus on if you make this false? I think we want at least 'heading', but as long as it makes sense for keyboard tabbing.

Reply:
I don't fully understand how the `autoFocus` parameter works, but setting it to `false` disabled the auto-focus on any fields by default, which is what I intended. If we set it to 'input', it automatically focuses on the "Segment" or "Values" field, which I find a bit unnatural.

Reply:
yeah 'input' was probably wrong, I guess false is fine as long as the focus is in the modal on the container or the header and tabbable to the form, but 'first-heading' would be a good default option also.